### PR TITLE
Dep upgrade

### PR DESCRIPTION
**Pull request by ar2rsawseen**

### CODE REVIEW

File: plugins/dbviewer/api/api.js [hunk 1]
```diff
@@ -243,18 +243,18 @@ var common = require('../../../api/utils/common.js'),
...
                                         }
-                                        done(false, true);
+                                        done();
                                     });
                                 }
                                 else {
-                                    done(false, true);
+                                    done();
                                 }
                             }, function(mapError) {
```
Comment (ronkorving) [inline plugins/dbviewer/api/api.js]: Since map became each, it's not a mapError anymore :) Nitpicking here ;)

File: package.json [hunk 2]
```diff
@@ -17,19 +17,19 @@
...
     "test": "grunt --verbose"
   },
   "dependencies": {
-    "argon2": "0.24.0",
-    "async": "2.6.3",
+    "argon2": "0.26.0",
+    "async": "3.2.0",
     "basic-auth": "2.0.1",
-    "bluebird": "3.5.5",
+    "bluebird": "3.7.2",
```
Comment (ronkorving) [inline package.json]: Oh nice, you have an `allSettled` now :)
https://github.com/petkaantonov/bluebird/releases/tag/v3.7.0

File: bin/upgrade/dev/upgrade_fs.sh [hunk 3]
```diff
@@ -5,13 +5,29 @@ echo "Running filesystem modifications"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
 CUR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+#upgrade nodejs
+if [ -f /etc/redhat-release ]; then
+	curl -sL https://rpm.nodesource.com/setup_10.x | bash -
```
Comment (ronkorving) [inline bin/upgrade/dev/upgrade_fs.sh]: Awesome :) I guess if we're officially Node 10 now, the Dockerfiles will need an update too

File: package.json (hunk 2 above)
Comment (ar2rsawseen) [reply package.json]: Interesting, but it is not in the docs :D
http://bluebirdjs.com/docs/api-reference.html

Comment (ronkorving) [reply package.json]: > Updated 06 Nov 2018 

Those docs don't see a lot of action :-/

**Pull request merged**